Tasks/XamarinAndroid/xamarinandroid.ts

```diff
@@ -47,7 +47,11 @@ else {
 //find xbuild location to use
 var xbuildToolPath = tl.which('xbuild');
 if(xbuildLocation) {
-    xbuildToolPath = xbuildLocation + '/xbuild';
+    xbuildToolPath = path.join(xbuildLocation, 'xbuild');
+    if (!tl.exist(xbuildToolPath)) {
+        xbuildToolPath = path.join(xbuildLocation, 'xbuild.exe');
```

Comment:
This was to handle running it on Windows or *nix. On Windows it has the extension .exe, but not on other OSes. Is there a better way?